index.js

```diff
@@ -104,9 +108,21 @@ function encodeBase64WithBtoa() {
   return btoa(unescape(encodeURIComponent(json)));
 }
 
+Converter.prototype.toURI = function () {
+  var json = this.toJSON();
+  return encodeURIComponent(json);
+};
+
 Converter.prototype.toComment = function (options) {
-  var base64 = this.toBase64();
-  var data = 'sourceMappingURL=data:application/json;charset=utf-8;base64,' + base64;
+  var encoding, content, data;
+  if (options && options.encoding === 'uri') {
```

Comment:
```suggestion
  if (options != null && options.encoding === 'uri') {
```
prefer to be explicit